nwb_explorer/nwb_model_interpreter/nwb_model_interpreter.py

```diff
@@ -19,31 +19,43 @@
 from .settings import *
 from ..utils import guessUnits
 
+import numpy as np
 from numpy import ndarray
 from pynwb import TimeSeries
 from pynwb.core import NWBBaseType
 from pynwb.core import LabelledDict
 from pynwb.file import Subject
 from pynwb.core import DynamicTable
 from pynwb.device import Device
-
+from pynwb.ophys import TwoPhotonSeries
+from h5py import Dataset
 
 class DefaultPyNWBToGeppettoMapper(object):
     ''' Extend this class to handle extra pynwb objects '''
 
     def __init__(self):
-        self._supported_types = ( TimeSeries )
+        self._supported_types = ( TwoPhotonSeries, TimeSeries )
```

Comment:
Why TwoPhotonSeries instead of ImageSeries?

Reply:
Fixed now